pom.xml

```diff
@@ -538,26 +539,26 @@
         <scope>provided</scope>
       </dependency>
       <dependency>
-        <groupId>com.google.gwt</groupId>
+        <groupId>org.gwtproject</groupId>
         <artifactId>gwt</artifactId>
```

Comment:
So gwt itself also needs to be added in build-dependencies?

Reply:
Yep, you right.
I will add it.